docs/src/content/docs/de/reference/configuration.mdx

```diff
@@ -131,7 +153,7 @@ Autogenerierte Untergruppen respektieren standardmäßig die Eigenschaft `collap
 sidebar: [
   // Eine zusammengefasste Gruppe von Links
   {
-    label: 'Collapsed Links',
+    label: 'Zusammengeklappte Links',
     collapsed: true,
     items: [
```

Comment:
This example seems to be missing the changes from https://github.com/withastro/starlight/pull/1874/files#diff-889e6ae5a36c15d03a30012e2936df94a63753b335bd5a8c56fac9a42e5c1dff and also proper highlights.